app/src/Components/ListTable/ListTableRowButtons.js

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
```

Comment:
Just as a general sidenote on this file: Wouldn't it be better, if `parameter` were just an object and you would then read from it in the switch case? Right now, `parameter` could either be a repository or an URL. If we add even more buttons in the future, it might get confusing, I think.

Reply:
instead i would want to think of how to rework the rowButton api to make it more clear. But i kind of want to leave it for a next PR...